### ISSUE TITLE
Contains an input of type undefined with both value and defaultValue props

### ISSUE THREAD

**Issue by kevinch** [thiagozanetti/react-intl-currency-input#9]
When i pass the `defaultValue` prop to set a value to the input i get this warning in the console:

```
warning.js?8a56:36 Warning: IntlCurrencyInput contains an input of type undefined 
with both value and defaultValue props. 
Input elements must be either controlled or uncontrolled (specify either the value prop, 
or the defaultValue prop, but not both).
```

**Comment by blissdev** [comment]
It looks like this works on master, but with this loaded through NPM it doesn't properly delete the `defaultValue` as implemented here: https://github.com/thiagozanetti/react-intl-currency-input/blob/master/src/index.jsx#L140

Could that be because of the indentation? Or maybe a new release is needed? (I think it is based on dates)

**Comment by thiagozanetti** [comment]
Seems fixed on 0.2.2
